lib/pbench/agent/tool_meister_start.py

```diff
@@ -660,7 +660,7 @@ def start(self, tm_dir: Path, full_hostname: str, logger: logging.Logger) -> Non
         bind_host_names = " ".join(bind_hostnames_l)
 
         # Create the Redis server pbench-specific configuration file
-        self.pid_file = tm_dir / f"redis_{self.bind_port:d}.pid"
+        self.pid_file = tm_dir / "redis.pid"
```

Comment:
I think that this means that a user cannot have TM-orchestrated Redis servers running on the same system, even if they are using different ports.  Isn't that a problem? (Again, should we be trying to detect collisions rather than just overwriting the PID file??)

Reply:
We always ensure the working directory used by the "create" orchestration path is unique.

Reply:
OK, so it's based on `benchmark_run_dir`, and anyone who runs two workloads out of the same directory at the same time is already in trouble.

Nevertheless, the PID file should not exist before we create it (right??), so it might be worth adding that check.

Reply:
Redis owns this behavior already.